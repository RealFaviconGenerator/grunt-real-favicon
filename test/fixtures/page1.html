<html>
  <head>
    <link rel="icon" href="some_icon.png">
  </head>
  <body>
    <h1>Page 1</h1>
    <p>Some content here</p>
  </body>
</html>
