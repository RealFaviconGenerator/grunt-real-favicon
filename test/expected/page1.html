<html>
  <head>
    
  <link rel="shortcut icon" href="/path/to/icons/favicon.ico">
<link rel="apple-touch-icon" sizes="57x57" href="/path/to/icons/apple-touch-icon-57x57.png">
<link rel="apple-touch-icon" sizes="114x114" href="/path/to/icons/apple-touch-icon-114x114.png">
<link rel="apple-touch-icon" sizes="72x72" href="/path/to/icons/apple-touch-icon-72x72.png">
<link rel="apple-touch-icon" sizes="144x144" href="/path/to/icons/apple-touch-icon-144x144.png">
<link rel="apple-touch-icon" sizes="60x60" href="/path/to/icons/apple-touch-icon-60x60.png">
<link rel="apple-touch-icon" sizes="120x120" href="/path/to/icons/apple-touch-icon-120x120.png">
<link rel="apple-touch-icon" sizes="76x76" href="/path/to/icons/apple-touch-icon-76x76.png">
<link rel="apple-touch-icon" sizes="152x152" href="/path/to/icons/apple-touch-icon-152x152.png">
<link rel="apple-touch-icon" sizes="180x180" href="/path/to/icons/apple-touch-icon-180x180.png">
<link rel="icon" type="image/png" href="/path/to/icons/favicon-192x192.png" sizes="192x192">
<link rel="icon" type="image/png" href="/path/to/icons/favicon-160x160.png" sizes="160x160">
<link rel="icon" type="image/png" href="/path/to/icons/favicon-96x96.png" sizes="96x96">
<link rel="icon" type="image/png" href="/path/to/icons/favicon-16x16.png" sizes="16x16">
<link rel="icon" type="image/png" href="/path/to/icons/favicon-32x32.png" sizes="32x32">
<meta name="msapplication-TileColor" content="#123456">
<meta name="msapplication-TileImage" content="/path/to/icons/mstile-144x144.png">
<meta name="msapplication-config" content="/path/to/icons/browserconfig.xml"></head>
  <body>
    <h1>Page 1</h1>
    <p>Some content here</p>
  </body>
</html>
